### ISSUE TITLE
Error "virtio_net: Could not create vmnet interface, permission denied or no entitlement?" running "sudo dlite daemon"

### ISSUE THREAD

**Issue by kesonno** [nlf/dlite#159]
Hi, this is my environment:
- OSX 10.10.5
- Fresh 2.0.0-beta5 installation (download binary from releases page)
- OSX firewall already disabled

Installation ends without problem, and after `dlite start` the file /etc/hosts is correctly modified, the IP can be pinged and /var/run/docker.sock is created.

But if I try to call every docker command (even the simplest `docker run hello-world`) I get the error

```
Error response from daemon: 400 Bad Request: malformed Host header
```

If I try to call `sudo dlite daemon` I get the error:

```
virtio_net: Could not create vmnet interface, permission denied or no entitlement?
```

and in the Console.app this messages appear:

```
04/04/16 03:34:21,150 InternetSharing[482]: MAC address associated with UUID 49EE8154-03FA-E511-81D0-84383555623E is already being used
04/04/16 03:34:21,150 InternetSharing[482]: unable to get MAC address
```

This is the content of my .dlite/config.json:

```
{
  "uuid": "5481ee49-fa03-11e5-81d0-84383555623e",
  "cpu_count": 4,
  "disk_size": 2,
  "memory": 2,
  "hostname": "local.docker",
  "dns_server": "192.168.64.1",
  "extra": "",
  "docker_version": "1.10.3",
  "route": false
}
```

Any thoughts?

**Comment by nlf** [comment]
make sure you `dlite stop` before you `sudo dlite daemon`, the error you're seeing there is because the vm is already running. try that and let's see what it does.

**Comment by synic** [comment]
I am having the same issue:

```
[~]$ dlite stop
Stopping the agent: ERROR!
The agent is already stopped
[~]$ sudo dlite daemon
virtio_net: Could not create vmnet interface, permission denied or no entitlement?
[~]$
```

Running OS X 10.11.4

**Comment by nlf** [comment]
run `ps aux | grep dlite` and see if the daemon didn't stop for some reason. if it didn't you can use the `kill -9` command with the pid to force it to stop

**Comment by synic** [comment]
Ah, that did it.  Thanks!  What is the password if I type `dlite ssh`  ?

**Comment by nlf** [comment]
it shouldn't ask for a password :/ sounds like the ssh setup failed. you can try to `ssh root@local.docker` with the password `dhyve` instead. if that fails, the output of `sudo dlite daemon` should show a device path at the top, something like `/dev/ttys00`. you can do `screen /dev/ttys00` and login as `root` with the password `dhyve` there

**Comment by synic** [comment]
Alright, I can get in, things still aren't working.  I didn't mean to take over this thread, I will ask in a new one.

**Comment by nlf** [comment]
@kesonno did any of the other chatter in here help you out? let me know if this is safe to close

**Comment by kesonno** [comment]
@nlf actually the problem causing the error in the issue subject was that `dlite stop` leave the process live, and I've resolved killing it manually. Thus, following the discussion in #160, I've downgraded DhyveOS to 3.0.0-beta4 and now the `dlite` process start and stop correctly, the vm start and I can login without password.

But trying to call every docker command, this error persist: 

```
Error response from daemon: 400 Bad Request: malformed Host header
```

Possibly related with docker/docker#20865?
